vendor/sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1_21_2/swagger.pb

```diff
@@ -36978,7 +36978,7 @@ containers
 enableServiceLinks�"�EnableServiceLinks indicates whether information about services should be injected into pod's environment variables, matching the syntax of Docker links. Optional: Defaults to true.�	
 boolean
 �
-serviceAccount�"sDeprecatedServiceAccount is a depreciated alias for ServiceAccountName. Deprecated: Use serviceAccountName instead.�
+serviceAccount�"sDeprecatedServiceAccount is a deprecated alias for ServiceAccountName. Deprecated: Use serviceAccountName instead.�
```

Comment:
Leave this to kustomize owners to fix in its repo. We cannot update vendor directly 😄

Reply:
seems like related to kustomize/kyaml doc?

Reply:
Woops, I missed that - should not be in this PR